src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationRepository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ProbationOfficerRepository
+
+@Service
+class ApplicationService(
+  private val probationOfficerRepository: ProbationOfficerRepository,
+  private val applicationRepository: ApplicationRepository
+) {
+  fun getAllApplications(userDistinguishedName: String): List<ApplicationEntity> {
```

Comment:
Could this be called `getAllApplicationsForUserName`? It's just clearer what we're doing then.